kubectl-cloudflow/config/config.go

```diff
@@ -142,19 +156,21 @@ func loadAndMergeConfigs(configFiles []string) (*configuration.Config, error) {
 			}
 		}
 	}
-	return config, nil
+	return &config, nil
 }
 
-func validateConfig(config *configuration.Config, applicationSpec cfapp.CloudflowApplicationSpec) error {
-	if config.IsEmpty() {
+func validateConfig(config *Config, applicationSpec cfapp.CloudflowApplicationSpec) error {
+	if config.isEmpty() {
```

Comment:
weird... an exported method name starting with smallcaps?

Reply:
it's not exported